syncer/streamer_controller.go

```diff
@@ -240,13 +243,18 @@ func (c *StreamerController) GetEvent(tctx *tcontext.Context) (event *replicatio
 	case *replication.RotateEvent:
 		// if is local binlog, binlog's name contain uuid information, need to save it
 		// if is remote binlog, need to add uuid information in binlog's name
-		if !c.setUUIDIfExists(string(ev.NextLogName)) {
+		c.Lock()
+		containUUID := c.setUUIDIfExists(string(ev.NextLogName))
+		uuidSuffix := c.uuidSuffix
+		c.Unlock()
+
+		if !containUUID {
 			if len(c.uuidSuffix) != 0 {
```

Comment:
Is `uuidSuffix` better than `c.uuidSuffix`?

Reply:
fix in 13b36c6